2025-03-20 16:48:28 info: API Gateway starting... 
2025-03-20 16:49:02 info: API Gateway starting... 
2025-03-20 19:43:07 info: User harish1 logged in 
2025-03-20 19:44:04 info: User harish1 logged in 
2025-03-24 17:38:46 error: Error logging in: userName is not defined 
2025-03-24 17:39:28 info: User harish logged in 
2025-03-24 17:42:12 error: uncaughtException: body is not defined
ReferenceError: body is not defined
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:3:25)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) ReferenceError: body is not defined
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:3:25)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:42:19 error: uncaughtException: body is not defined
ReferenceError: body is not defined
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:3:25)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) ReferenceError: body is not defined
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:3:25)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:42:39 error: uncaughtException: body is not defined
ReferenceError: body is not defined
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:45)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) ReferenceError: body is not defined
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:45)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:42:41 error: uncaughtException: body is not defined
ReferenceError: body is not defined
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:3:25)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) ReferenceError: body is not defined
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:3:25)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:42:44 error: uncaughtException: Unexpected token '='
C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:1
const  = require('express-validator');
       ^

SyntaxError: Unexpected token '='
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:1
const  = require('express-validator');
       ^

SyntaxError: Unexpected token '='
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:45:20 error: uncaughtException: Unexpected identifier 'next'
C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:13
  next();
  ^^^^

SyntaxError: Unexpected identifier 'next'
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:13
  next();
  ^^^^

SyntaxError: Unexpected identifier 'next'
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:45:34 error: Error logging in: Cannot set headers after they are sent to the client 
2025-03-24 17:45:34 error: unhandledRejection: Cannot set headers after they are sent to the client
Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (node:_http_outgoing:659:11)
    at ServerResponse.header (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:794:10)
    at ServerResponse.send (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:174:12)
    at ServerResponse.json (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:278:15)
    at login (C:\Users\haris\Documents\system design\auth_service\controllers\Authcontroller.js:55:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (node:_http_outgoing:659:11)
    at ServerResponse.header (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:794:10)
    at ServerResponse.send (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:174:12)
    at ServerResponse.json (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:278:15)
    at login (C:\Users\haris\Documents\system design\auth_service\controllers\Authcontroller.js:55:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2025-03-24 17:45:36 error: Error logging in: Cannot set headers after they are sent to the client 
2025-03-24 17:45:36 error: unhandledRejection: Cannot set headers after they are sent to the client
Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (node:_http_outgoing:659:11)
    at ServerResponse.header (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:794:10)
    at ServerResponse.send (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:174:12)
    at ServerResponse.json (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:278:15)
    at login (C:\Users\haris\Documents\system design\auth_service\controllers\Authcontroller.js:55:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (node:_http_outgoing:659:11)
    at ServerResponse.header (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:794:10)
    at ServerResponse.send (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:174:12)
    at ServerResponse.json (C:\Users\haris\Documents\system design\auth_service\node_modules\express\lib\response.js:278:15)
    at login (C:\Users\haris\Documents\system design\auth_service\controllers\Authcontroller.js:55:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2025-03-24 17:45:36 error: uncaughtException: write after end
Error: write after end
    at writeAfterEnd (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:264:12)
    at Writable.write (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:300:21)
    at File.log (C:\Users\haris\Documents\system design\auth_service\node_modules\winston\lib\winston\transports\file.js:222:34)
    at RejectionStream._write (C:\Users\haris\Documents\system design\auth_service\node_modules\winston\lib\winston\rejection-stream.js:46:29)
    at doWrite (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:390:139)
    at writeOrBuffer (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:381:5)
    at Writable.write (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:302:11)
    at DerivedLogger.ondata (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_readable.js:629:20)
    at DerivedLogger.emit (node:events:531:35)
    at addChunk (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_readable.js:279:12) Error: write after end
    at writeAfterEnd (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:264:12)
    at Writable.write (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:300:21)
    at File.log (C:\Users\haris\Documents\system design\auth_service\node_modules\winston\lib\winston\transports\file.js:222:34)
    at RejectionStream._write (C:\Users\haris\Documents\system design\auth_service\node_modules\winston\lib\winston\rejection-stream.js:46:29)
    at doWrite (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:390:139)
    at writeOrBuffer (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:381:5)
    at Writable.write (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_writable.js:302:11)
    at DerivedLogger.ondata (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_readable.js:629:20)
    at DerivedLogger.emit (node:events:531:35)
    at addChunk (C:\Users\haris\Documents\system design\auth_service\node_modules\readable-stream\lib\_stream_readable.js:279:12)
2025-03-24 17:48:11 error: uncaughtException: body(...).isEmail(...).isemwithMessage is not a function
TypeError: body(...).isEmail(...).isemwithMessage is not a function
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:27)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) TypeError: body(...).isEmail(...).isemwithMessage is not a function
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:27)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:48:13 error: uncaughtException: body(...).isEmail(...).isEmptywithMessage is not a function
TypeError: body(...).isEmail(...).isEmptywithMessage is not a function
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:27)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) TypeError: body(...).isEmail(...).isEmptywithMessage is not a function
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:27)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:53:08 error: uncaughtException: body(...).isEmail(...).notwithMessage is not a function
TypeError: body(...).isEmail(...).notwithMessage is not a function
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:27)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) TypeError: body(...).isEmail(...).notwithMessage is not a function
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:27)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:53:47 error: uncaughtException: Unexpected token '.'
C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4
  body('email').isEmail().notEmpty()..bail().withMessage('Invalid email address'),
                                     ^

SyntaxError: Unexpected token '.'
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4
  body('email').isEmail().notEmpty()..bail().withMessage('Invalid email address'),
                                     ^

SyntaxError: Unexpected token '.'
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:54:28 error: uncaughtException: body(...).isEmail(...).notEmpty(...).withMessage(...).trimbail is not a function
TypeError: body(...).isEmail(...).notEmpty(...).withMessage(...).trimbail is not a function
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:71)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) TypeError: body(...).isEmail(...).notEmpty(...).withMessage(...).trimbail is not a function
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:4:71)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 17:58:11 error: uncaughtException: missing ) after argument list
C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:7
  .notEmpty().withMessage('Email can't be empty').bail()
                          ^^^^^^^^^^^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:7
  .notEmpty().withMessage('Email can't be empty').bail()
                          ^^^^^^^^^^^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 18:04:40 info: User harish1 created 
2025-03-24 18:05:45 error: uncaughtException: Invalid or unexpected token
C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:25
    .withMessage'),
                ^^^

SyntaxError: Invalid or unexpected token
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:49)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10) C:\Users\haris\Documents\system design\auth_service\validations\authValidations.js:25
    .withMessage'),
                ^^^

SyntaxError: Invalid or unexpected token
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\haris\Documents\system design\auth_service\routes\AuthRoutes.js:3:49)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
2025-03-24 18:06:25 info: User harish1 created 
2025-03-24 18:06:42 info: User harish1 logged in 
2025-03-24 18:10:43 info: User harish1 logged in 
2025-03-24 19:12:34 info: Server is running on port 4001 
2025-07-03 13:44:49 error: unhandledRejection: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
MongooseError: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
    at NativeConnection.createClient (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\drivers\node-mongodb-native\connection.js:236:11)
    at NativeConnection.openUri (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\connection.js:1071:34)
    at Mongoose.connect (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\mongoose.js:446:15)
    at Object.<anonymous> (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\db.js:4:10)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24) MongooseError: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
    at NativeConnection.createClient (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\drivers\node-mongodb-native\connection.js:236:11)
    at NativeConnection.openUri (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\connection.js:1071:34)
    at Mongoose.connect (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\mongoose.js:446:15)
    at Object.<anonymous> (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\db.js:4:10)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
2025-07-03 13:48:03 error: unhandledRejection: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
MongooseError: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
    at NativeConnection.createClient (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\drivers\node-mongodb-native\connection.js:236:11)
    at NativeConnection.openUri (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\connection.js:1071:34)
    at Mongoose.connect (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\mongoose.js:446:15)
    at Object.<anonymous> (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\db.js:4:10)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24) MongooseError: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
    at NativeConnection.createClient (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\drivers\node-mongodb-native\connection.js:236:11)
    at NativeConnection.openUri (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\connection.js:1071:34)
    at Mongoose.connect (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\node_modules\mongoose\lib\mongoose.js:446:15)
    at Object.<anonymous> (C:\Users\haris\Pictures\sunotal-farms-services\auth_service\db.js:4:10)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
2025-07-03 17:24:48 error: uncaughtException: Cannot find module '../database/models/AuthModel'
Require stack:
- C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js
- C:\Users\haris\Videos\micro-services\auth_service\src\routes\AuthRoutes.js
- C:\Users\haris\Videos\micro-services\auth_service\src\expressApp.js
- C:\Users\haris\Videos\micro-services\auth_service\src\server.js
Error: Cannot find module '../database/models/AuthModel'
Require stack:
- C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js
- C:\Users\haris\Videos\micro-services\auth_service\src\routes\AuthRoutes.js
- C:\Users\haris\Videos\micro-services\auth_service\src\expressApp.js
- C:\Users\haris\Videos\micro-services\auth_service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js:4:15)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32) Error: Cannot find module '../database/models/AuthModel'
Require stack:
- C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js
- C:\Users\haris\Videos\micro-services\auth_service\src\routes\AuthRoutes.js
- C:\Users\haris\Videos\micro-services\auth_service\src\expressApp.js
- C:\Users\haris\Videos\micro-services\auth_service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js:4:15)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
2025-07-03 17:25:01 error: uncaughtException: Cannot find module '../'
Require stack:
- C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js
- C:\Users\haris\Videos\micro-services\auth_service\src\routes\AuthRoutes.js
- C:\Users\haris\Videos\micro-services\auth_service\src\expressApp.js
- C:\Users\haris\Videos\micro-services\auth_service\src\server.js
Error: Cannot find module '../'
Require stack:
- C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js
- C:\Users\haris\Videos\micro-services\auth_service\src\routes\AuthRoutes.js
- C:\Users\haris\Videos\micro-services\auth_service\src\expressApp.js
- C:\Users\haris\Videos\micro-services\auth_service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js:4:15)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32) Error: Cannot find module '../'
Require stack:
- C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js
- C:\Users\haris\Videos\micro-services\auth_service\src\routes\AuthRoutes.js
- C:\Users\haris\Videos\micro-services\auth_service\src\expressApp.js
- C:\Users\haris\Videos\micro-services\auth_service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Videos\micro-services\auth_service\src\controllers\Authcontroller.js:4:15)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
2025-07-03 19:30:15 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:40:12 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:41:39 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:41:56 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:44:13 error: uncaughtException: Unexpected token '}'
C:\Users\haris\Videos\micro-services\auth_service\src\database\repositories\userAuthRepository.js:54
        } catch (error) {
        ^

SyntaxError: Unexpected token '}'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Videos\micro-services\auth_service\src\database\index.js:3:25) C:\Users\haris\Videos\micro-services\auth_service\src\database\repositories\userAuthRepository.js:54
        } catch (error) {
        ^

SyntaxError: Unexpected token '}'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Videos\micro-services\auth_service\src\database\index.js:3:25)
2025-07-03 19:44:59 error: uncaughtException: Unexpected token '}'
C:\Users\haris\Videos\micro-services\auth_service\src\database\repositories\userAuthRepository.js:70
        } catch (error) {
        ^

SyntaxError: Unexpected token '}'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Videos\micro-services\auth_service\src\database\index.js:3:25) C:\Users\haris\Videos\micro-services\auth_service\src\database\repositories\userAuthRepository.js:70
        } catch (error) {
        ^

SyntaxError: Unexpected token '}'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Videos\micro-services\auth_service\src\database\index.js:3:25)
2025-07-03 19:46:22 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:46:22 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:46:22 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:46:23 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:46:23 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:46:23 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:46:23 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:46:23 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:46:23 error: Error logging in: Users.findOne is not a function 
2025-07-03 19:46:23 error: Error logging in: Users.findOne is not a function 
2025-07-04 11:53:28 info: User harish created 
2025-07-04 12:02:38 info: User harish logged in 
2025-07-04 12:02:52 info: User harish logged in 
2025-07-04 12:07:12 info: User harish logged in 
2025-07-04 12:07:13 info: User harish logged in 
2025-07-04 12:09:43 info: User harish logged in 
2025-07-04 12:12:18 info: User harish logged in 
2025-07-04 12:14:06 info: User harish logged in 
2025-07-10 11:40:03 error: unhandledRejection: Cannot access 'PORT' before initialization
ReferenceError: Cannot access 'PORT' before initialization
    at startServer (C:\Users\haris\Pictures\backend-services\auth-service\src\server.js:13:18)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5) ReferenceError: Cannot access 'PORT' before initialization
    at startServer (C:\Users\haris\Pictures\backend-services\auth-service\src\server.js:13:18)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-10 18:38:24 error: uncaughtException: Invalid left-hand side in assignment
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:12
            user?.cartItems = [];
            ^^^^^^^^^^^^^^^

SyntaxError: Invalid left-hand side in assignment
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:2:62) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:12
            user?.cartItems = [];
            ^^^^^^^^^^^^^^^

SyntaxError: Invalid left-hand side in assignment
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:2:62)
2025-07-10 18:38:39 error: unhandledRejection: Router.use() requires a middleware function but got a Object
TypeError: Router.use() requires a middleware function but got a Object
    at Function.use (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:469:13)
    at Function.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\application.js:227:21)
    at Array.forEach (<anonymous>)
    at Function.use (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\application.js:224:7)
    at module.exports (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:14:9)
    at startServer (C:\Users\haris\Pictures\backend-services\auth-service\src\server.js:11:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5) TypeError: Router.use() requires a middleware function but got a Object
    at Function.use (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:469:13)
    at Function.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\application.js:227:21)
    at Array.forEach (<anonymous>)
    at Function.use (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\application.js:224:7)
    at module.exports (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:14:9)
    at startServer (C:\Users\haris\Pictures\backend-services\auth-service\src\server.js:11:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-10 18:42:31 info: User harish1 created 
2025-07-10 18:43:35 info: User harish1 logged in 
2025-07-10 18:43:56 info: User harish1 logged in 
2025-07-10 18:49:20 error: uncaughtException: Route.get() requires a callback function but got a [object Undefined]
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12) Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
2025-07-10 18:49:35 error: uncaughtException: Route.get() requires a callback function but got a [object Undefined]
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12) Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
2025-07-10 18:49:40 error: uncaughtException: Route.get() requires a callback function but got a [object Undefined]
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12) Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
2025-07-10 18:49:41 error: unhandledRejection: Router.use() requires a middleware function but got a Object
TypeError: Router.use() requires a middleware function but got a Object
    at Function.use (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:469:13)
    at Function.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\application.js:227:21)
    at Array.forEach (<anonymous>)
    at Function.use (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\application.js:224:7)
    at module.exports (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:14:9)
    at startServer (C:\Users\haris\Pictures\backend-services\auth-service\src\server.js:11:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5) TypeError: Router.use() requires a middleware function but got a Object
    at Function.use (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:469:13)
    at Function.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\application.js:227:21)
    at Array.forEach (<anonymous>)
    at Function.use (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\application.js:224:7)
    at module.exports (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:14:9)
    at startServer (C:\Users\haris\Pictures\backend-services\auth-service\src\server.js:11:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-10 18:49:42 error: uncaughtException: Route.get() requires a callback function but got a [object Undefined]
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12) Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
2025-07-10 18:50:04 error: uncaughtException: Route.get() requires a callback function but got a [object Undefined]
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12) Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
2025-07-10 18:55:10 error: uncaughtException: Cannot find module './'
Require stack:
- C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\server.js
Error: Cannot find module './'
Require stack:
- C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:1:23)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32) Error: Cannot find module './'
Require stack:
- C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:1:23)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
2025-07-10 18:55:12 error: uncaughtException: Cannot find module '../'
Require stack:
- C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\server.js
Error: Cannot find module '../'
Require stack:
- C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:1:23)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32) Error: Cannot find module '../'
Require stack:
- C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:1:23)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
2025-07-10 19:16:25 error: uncaughtException: removeProductFromCart is not defined
ReferenceError: removeProductFromCart is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:67:2)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) ReferenceError: removeProductFromCart is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:67:2)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-10 19:16:29 error: uncaughtException: Route.delete() requires a callback function but got a [object Undefined]
Error: Route.delete() requires a callback function but got a [object Undefined]
    at Route.<computed> [as delete] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as delete] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:14:14)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12) Error: Route.delete() requires a callback function but got a [object Undefined]
    at Route.<computed> [as delete] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as delete] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:14:14)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
2025-07-11 09:40:34 error: uncaughtException: getAllUsers is not defined
ReferenceError: getAllUsers is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:28)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getAllUsers is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:28)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-11 09:40:42 error: uncaughtException: getAllUsers is not defined
ReferenceError: getAllUsers is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:11:28)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getAllUsers is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:11:28)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-11 17:45:07 error: uncaughtException: block is not defined
ReferenceError: block is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:35)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: block is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:35)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-11 17:48:10 error: uncaughtException: unbl is not defined
ReferenceError: unbl is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:13:37)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: unbl is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:13:37)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-12 11:05:02 info: User harish1 created 
2025-07-12 11:05:07 info: User harish1 created 
2025-07-12 11:09:55 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:87:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:105) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:87:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:105)
2025-07-12 11:09:58 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:89:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:105) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:89:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:105)
2025-07-12 11:09:59 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:88:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:105) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:88:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:105)
2025-07-12 11:10:15 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-12 11:10:16 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-12 11:10:17 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-12 11:10:22 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-12 11:10:23 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-12 11:10:25 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-12 11:10:30 error: uncaughtException: getUserStats is not defined
ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:31)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUserStats is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:31)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-15 09:34:17 error: uncaughtException: async is not defined
ReferenceError: async is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:70:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:7:5) ReferenceError: async is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:70:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:7:5)
2025-07-15 09:34:18 error: uncaughtException: async is not defined
ReferenceError: async is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:70:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:7:5) ReferenceError: async is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:70:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:7:5)
2025-07-15 09:34:56 error: uncaughtException: Unexpected identifier 'form'
C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:15
remove form cart
       ^^^^

SyntaxError: Unexpected identifier 'form'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:5:20) C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:15
remove form cart
       ^^^^

SyntaxError: Unexpected identifier 'form'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:5:20)
2025-07-15 09:35:01 error: uncaughtException: Unexpected identifier 'form'
C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:15
remove form cart
       ^^^^

SyntaxError: Unexpected identifier 'form'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:5:20) C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:15
remove form cart
       ^^^^

SyntaxError: Unexpected identifier 'form'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:5:20)
2025-07-15 09:35:04 error: uncaughtException: Unexpected identifier 'form'
C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:16
remove form cart
       ^^^^

SyntaxError: Unexpected identifier 'form'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:5:20) C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:16
remove form cart
       ^^^^

SyntaxError: Unexpected identifier 'form'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:5:20)
2025-07-15 09:49:40 error: uncaughtException: Unexpected token 'if'
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:29
		if (!user) return res.status(404).json({ message: 'User not found' })
		^^

SyntaxError: Unexpected token 'if'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:29
		if (!user) return res.status(404).json({ message: 'User not found' })
		^^

SyntaxError: Unexpected token 'if'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:49:50 error: uncaughtException: Unexpected token 'return'
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:29
		if (!user) return res.status(404).json({ message: 'User not found' })
		           ^^^^^^

SyntaxError: Unexpected token 'return'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:29
		if (!user) return res.status(404).json({ message: 'User not found' })
		           ^^^^^^

SyntaxError: Unexpected token 'return'
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:49:54 error: uncaughtException: missing ) after argument list
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(http://localhost:5000/api/products/get-product/686a7794d639ea8e9b4fd439)
                                    ^^^^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(http://localhost:5000/api/products/get-product/686a7794d639ea8e9b4fd439)
                                    ^^^^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:49:55 error: uncaughtException: missing ) after argument list
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(http://localhost:5000/api/products/get-product/686a7794d639ea8e9b4fd439);
                                    ^^^^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(http://localhost:5000/api/products/get-product/686a7794d639ea8e9b4fd439);
                                    ^^^^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:49:58 error: uncaughtException: missing ) after argument list
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(http://localhost:5000/api/products/get-product/686a7794d639ea8e9b4fd439');
                                    ^^^^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(http://localhost:5000/api/products/get-product/686a7794d639ea8e9b4fd439');
                                    ^^^^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:51:35 error: uncaughtException: Unexpected end of input
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:91



SyntaxError: Unexpected end of input
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:91



SyntaxError: Unexpected end of input
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:51:57 error: uncaughtException: Invalid or unexpected token
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get($/api/products/get-product/686a7794d639ea8e9b4fd439`);
                                                               ^^^

SyntaxError: Invalid or unexpected token
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get($/api/products/get-product/686a7794d639ea8e9b4fd439`);
                                                               ^^^

SyntaxError: Invalid or unexpected token
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:52:00 error: uncaughtException: missing ) after argument list
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(${API_GATEWAY_URL/api/products/get-product/686a7794d639ea8e9b4fd439`);
                                    ^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(${API_GATEWAY_URL/api/products/get-product/686a7794d639ea8e9b4fd439`);
                                    ^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:52:01 error: uncaughtException: missing ) after argument list
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(${API_GATEWAY_URL/api/products/get-product/686a7794d639ea8e9b4fd439`);
                                    ^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:28
        const res = await axios.get(${API_GATEWAY_URL/api/products/get-product/686a7794d639ea8e9b4fd439`);
                                    ^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:52:05 error: uncaughtException: missing ) after argument list
C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:29
        const res = await axios.get(${API_GATEWAY_URL}/api/products/get-product/686a7794d639ea8e9b4fd439`);
                                    ^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5) C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:29
        const res = await axios.get(${API_GATEWAY_URL}/api/products/get-product/686a7794d639ea8e9b4fd439`);
                                    ^

SyntaxError: missing ) after argument list
    at wrapSafe (node:internal/modules/cjs/loader:1486:18)
    at Module._compile (node:internal/modules/cjs/loader:1528:20)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js:8:5)
2025-07-15 09:54:13 error: uncaughtException: Cannot find module 'axios'
Require stack:
- C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\server.js
Error: Cannot find module 'axios'
Require stack:
- C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:1:15)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32) Error: Cannot find module 'axios'
Require stack:
- C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\routes\cartRoutes.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js
- C:\Users\haris\Pictures\backend-services\auth-service\src\server.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\cartController.js:1:15)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
2025-07-17 15:35:57 error: uncaughtException: getUser is not defined
ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:26)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:26)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-17 15:36:07 error: uncaughtException: getUser is not defined
ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:28)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:28)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-17 15:36:08 error: uncaughtException: getUser is not defined
ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:34)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:34)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
2025-07-17 15:36:23 error: uncaughtException: Route.post() requires a callback function but got a [object Undefined]
Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as post] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12) Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as post] (C:\Users\haris\Pictures\backend-services\auth-service\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:12:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
2025-07-17 15:36:28 error: uncaughtException: getUser is not defined
ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:94:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:128) ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:94:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:128)
2025-07-17 15:36:32 error: uncaughtException: getUser is not defined
ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:96:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:128) ReferenceError: getUser is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\controllers\Authcontroller.js:96:5)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:2:128)
2025-07-17 16:32:02 error: uncaughtException: addUserAddress is not defined
ReferenceError: addUserAddress is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:42)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20) ReferenceError: addUserAddress is not defined
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\routes\AuthRoutes.js:14:42)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Users\haris\Pictures\backend-services\auth-service\src\expressApp.js:4:20)
